python/cudf/cudf/tests/test_rolling.py

```diff
@@ -162,3 +162,40 @@ def test_rolling_getitem_window():
     pdf = pd.DataFrame({"x": np.arange(len(index))}, index=index)
     gdf = cudf.from_pandas(pdf)
     assert_eq(pdf.rolling("2h").x.mean(), gdf.rolling("2h").x.mean())
+
+
+@pytest.mark.parametrize("data,index", [([1, 4, 5, 2, 9, 7], None)])
+@pytest.mark.parametrize("nulls", ["none"])
+@pytest.mark.parametrize("center", [True, False])
+def test_rollling_series_numba_udf(data, index, nulls, center):
+    if len(data) > 0:
+        if nulls == "one":
```

Comment:
We should just get rid of the `nulls` parameter and this `if-else` block for this test